src/event_pusher/mod_event_pusher_push_plugin_defaults.erl

```diff
@@ -122,3 +125,16 @@ make_form_field({Name, Value}) ->
            attrs = [{<<"var">>, Name}],
            children = [#xmlel{name = <<"value">>, children = [#xmlcdata{content = Value}]}]}.
 
+get_unread_count(Acc, From, To) ->
+            {_, Host} = jid:to_lus(To),
+            Acc0 = ejabberd_hooks:run_fold(inbox_unread_count, Host, Acc, [To]),
+            UnreadCount = try mongoose_acc:get(inbox, unread_count, Acc0) of
+                              Val ->
+                                  Val
+                          catch
+                              _:_ ->
+                                  1
+                          end,
+            {Acc0,
+             integer_to_binary(UnreadCount)}.
```

Comment:
I wonder if we need the `UnreadCount` as a binary here or can we convert to binary later, when the push notification is being created?